operator/cmd/flags.go

```diff
@@ -330,3 +325,20 @@ func init() {
 
 	Vp.BindPFlags(flags)
 }
+
+// operatorPprofConfig holds the configuration for the operator pprof cell.
+// Differently from the agent and the clustermesh-apiserver, the operator prefixes
+// the pprof related flags with the string "operator-".
+// To reuse the same cell, we need a different config type to map the same fields
+// to the operator-specific pprof flag names.
+type operatorPprofConfig struct {
+	OperatorPprof        bool
+	OperatorPprofAddress string
+	OperatorPprofPort    int
+}
+
+func (def operatorPprofConfig) Flags(flags *pflag.FlagSet) {
```

Comment:
Should this be a pointer receiver?

Reply:
Since there's no need to modify the receiver and copying an `operatorPprofConfig` instance is very cheap (besides, this method should be called just once at startup), I think it is fine to use a value receiver.

Reply:
sounds good.